### PR TITLE
Extend propono configuration

### PR DESCRIPTION
**Pull request by stevenbarragan**
It adds the possibility of configure all propono config variables from the `propono.yml`, like `max_retries`, `tcp_host`, `tcp_port`, `udp_host`, `udp_port`.

### CODE REVIEW

File: lib/larva/configurator.rb [hunk 1]
```diff
@@ -43,7 +33,22 @@ def parse_config_file(filename)
...
+      Hash.new()
+    end
+
+    def propono_setup(propono_config)
+      Propono.config do |config|
+        # Backwards compatible with region key instead of queue_region
+        region              = propono_config.delete(:region)
+        config.queue_region = region if region
+
+        config.udp_host     = "pergo.meducation.net"
```
Comment (stevenbarragan) [inline lib/larva/configurator.rb]: I think this configuration should come form the `propono.yml` file as well.

**Comment by iHiD** [comment]
Hey @stevenbarragan, thanks for this.

**1)** Fancy removing these lines?
```
config.udp_host     = "pergo.meducation.net"
config.udp_port     = "9732"
```

These are here from a legacy version of the library, which was scoped to Meducation pre-opensourcing it.

**2)** Can you see any issues compressing
```
region              = propono_config.delete(:region)
config.queue_region = region if region
```

into 
```
config.queue_region = propono_config.delete(:region)
```
?

Thanks again! :blue_heart:

**Comment by stevenbarragan** [comment]
wow, that was a fast response.

1. removing `udp_host` and `udp_port` defaults.
2. you're right about the little refactor. I don't see why not.

**Comment by iHiD** [comment]
Ha. My responses tend to be either instant or earth-rotatingly slow, depending on whether I'm in developer-mode or entrepreneur-mode when stuff happens :)

That's great, thanks. I'll merge and release a new version once Travis is done.

**Comment by iHiD** [comment]
@stevenbarragan 1.3.0 released. Thank you! :blue_heart:

**Comment by stevenbarragan** [comment]
No problem, thanks for the fast response.

**Pull request merged**